### DISCUSSION TITLE
trouble connecting to the server

### DISCUSSION THREAD

**Discussion by v-collin** [pterodactyl/panel#3129] [Support] [answered]
I recently moved to a new house with a new router so i got an new ip address.
now i exactly copied every port from the old router to port forward and i changed my dns settings of my domain.
everything network wise is now connected propperly.
i renewed my ssl certificate succesfully.
i changed my allocations succesfully to the new ip address.
now when i try to connect to my server's console it gives me this [error](https://gyazo.com/d8a12632dce8f7cde65caa2dc9aefc97)
[this](https://ptero.co/xokuyiroze) is my wings diagnostics.
[this](http://bin.ptdl.co/pwoii) is my panel log.
I am not getting any response from the discord server so i decided to try it here.

**Comment by v-collin** [comment]
CURL https://panel.venetius.nl:8080 returns this:
{"error":"The required authorization heads were not present in the request."}root@pterodactyl:~#

**Comment by trenutoo** [comment, marked as answer]
Didn't we resolve this in the Discord yesterday or is this still relevant?

**Reply by v-collin** [reply]
this is already resolved

**Reply by Vapok** [reply]
This is great that you found the answer and marked _this_ as the answer. But for someone like me who says, "hey look, someone has the same issue I do!"..

this is not really an answer that should be marked as one. Just sayin'.  Having the same issue.  Any help would be appreciated.

**Reply by trenutoo** [reply]
> This is great that you found the answer and marked _this_ as the answer. But for someone like me who says, "hey look, someone has the same issue I do!"..
> 
> this is not really an answer that should be marked as one. Just sayin'.  Having the same issue.  Any help would be appreciated.

You should join the Discord server for support and read wings troubleshooting documentation.

**Reply by robisodd** [reply]
I just joined the Discord server.  I do not see a "Support and Read Wings Troubleshooting Documentation" channel.  I would be helpful to post the answer here, or other locations that are searchable and not on the deep web.

**Comment by Pabsb** [comment]
How was this fixed? Troubleshooting documentation went through all of it to no avail?

**Reply by fucksophie** [reply]
This was solved by the discord bot responding with that the port that Collin used was already being used by something else, so checking via `netstat -tulpn | grep <port>` and then removing that service.
